.env.example

```diff
@@ -8,4 +8,13 @@ WALLET_KIT_MT_NAME=
 WALLET_KIT_MT_DESCRIPTION=
 WALLET_KIT_MT_URL=
 WALLET_KIT_MT_ICON=
-WALLET_KIT_MT_REDIRECT_NATIVE=
\ No newline at end of file
+WALLET_KIT_MT_REDIRECT_NATIVE=
+
+SENTRY_DSN=
+
+ANDROID_DEBUG_KEYSTORE_PASSWORD=
+ANDROID_DEBUG_KEYSTORE_ALIAS=
+ANDROID_DEV_KEYSTORE_PASSWORD=
+ANDROID_DEV_KEYSTORE_ALIAS=
+ANDROID_PROD_KEYSTORE_PASSWORD=
+ANDROID_PROD_KEYSTORE_ALIAS=
```

Comment:
Added `env vars` to set Android `keystore` alias and passwords.

You should be able to download the keystore files from the 1Password shared Wallet Eng folder or else get in contact with me through keybase to get them.

Please note that the `release.keystore` file has been renamed to `prod-release.keystore` and now we also have a `dev-release.keystore` file.